src/rpa_ri_gpw.F

```diff
@@ -3592,14 +3282,15 @@ SUBROUTINE rpa_num_int(qs_env, Erpa, mp2_env, para_env, para_env_RPA, para_env_s
             IF (do_ri_sos_laplace_mp2) THEN
                ! calculate the trace of the product Q*Q
                trace_XX = 0.0_dp
-               IF (my_open_shell) THEN
-                  DO jjB = 1, ncol_local
-                     trace_XX = trace_XX+DOT_PRODUCT(fm_mat_Q%local_data(:, jjB), fm_mat_Q_beta%local_data(:, jjB))
+               DO jjB = 1, ncol_local
```

Comment:
Why do you replace that code with older code?

Reply:
Sorry... Is reverted.